core/dbt/graph/selector_methods.py

```diff
@@ -230,6 +231,16 @@ def search(self, included_nodes: Set[UniqueId], selector: str) -> Iterator[Uniqu
                 yield node
 
 
+class AccessSelectorMethod(SelectorMethod):
+    def search(self, included_nodes: Set[UniqueId], selector: str) -> Iterator[UniqueId]:
+        """yields nodes from included in the specified group"""
```

Comment:
looks like a copy-pasta typo: should probably be "yields nodes from included with the specified access"